packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -361,9 +744,61 @@ export class ApexDebug extends DebugSession {
           const sourcePath = this.myBreakpointService.getSourcePathFromTyperef(
             serverFrames[i].typeRef
           );
+          const frameInfo = new ApexDebugStackFrameInfo(
+            requestId,
+            serverFrames[i].frameNumber
+          );
+          const frameId = this.stackFrameInfos.create(frameInfo);
+          if (i === 0 && stateRespObj.stateResponse.state) {
```

Comment:
You changed StackFrame id below from i to frameId. Should this be frameId as well?

Reply:
No. This needs to identify the first frame only. The state response includes variables information for the first frame (index == 0).